bot_modules/config_manager/index.js

```diff
@@ -45,7 +45,7 @@ exports.init = function init (client) {
       if (args[1] === 'edit') {
         if (args[2] === 'all') {
           try {
-            configEditor.set(JSON.parse(args.slice(3).join(' ')))
+            exports.configEditor.set(JSON.parse(args.slice(3).join(' ')))
```

Comment:
Could `exports.configEditor` not be replaced with `this`

Reply:
Not exactly, because this code isn't part of `configEditor`, so `this` isn't `configEditor`

Reply:
The current just doesn't seem clean, but likely the best way. Thanks for checking.